packages/apps/plugins/plugin-kanban/src/components/KanbanMain.tsx

```diff
@@ -31,9 +32,8 @@ export const KanbanMain: FC<{ data: [SpaceProxy, KanbanType] }> = ({ data: [spac
       ),
   };
 
-  // TODO(burdon): Style/color standards for panels, borders, text, etc.
   return (
-    <Main.Content classNames='flex flex-col grow min-bs-[100vh] overflow-hidden bg-white dark:bg-neutral-925'>
+    <Main.Content classNames={mx('flex flex-col grow min-bs-[100vh] overflow-hidden', Styles.level0.bg)}>
```

Comment:
Please avoid using `mx` in `classNames`, it’s redundant. Instead:
`<Main.Content classNames={['flex flex-col grow min-bs-[100vh] overflow-hidden', Styles.level0.bg]}>`